packages/pigeon/platform_tests/test_plugin/macos/Classes/CoreTests.gen.swift

```diff
@@ -122,24 +129,25 @@ struct AllNullableTypes {
   var aNullableString: String? = nil
 
   static func fromList(_ list: [Any]) -> AllNullableTypes? {
-    let aNullableBool = list[0] as! Bool? 
-    let aNullableInt = list[1] as! Int64? 
-    let aNullableInt64 = list[2] as! Int64? 
-    let aNullableDouble = list[3] as! Double? 
-    let aNullableByteArray = list[4] as! FlutterStandardTypedData? 
-    let aNullable4ByteArray = list[5] as! FlutterStandardTypedData? 
-    let aNullable8ByteArray = list[6] as! FlutterStandardTypedData? 
-    let aNullableFloatArray = list[7] as! FlutterStandardTypedData? 
-    let aNullableList = list[8] as! [Any]? 
-    let aNullableMap = list[9] as! [AnyHashable: Any]? 
-    let nullableNestedList = list[10] as! [[Bool?]?]? 
-    let nullableMapWithAnnotations = list[11] as! [String?: String?]? 
-    let nullableMapWithObject = list[12] as! [String?: Any?]? 
+    let aNullableBool = (nilOrValue(value: list[0]) as Any) as! Bool? 
+    let aNullableInt = (nilOrValue(value: list[1]) as Any) as! Int64? 
+    let aNullableInt64 = (nilOrValue(value: list[2]) as Any) as! Int64? 
+    let aNullableDouble = (nilOrValue(value: list[3]) as Any) as! Double? 
+    let aNullableByteArray = (nilOrValue(value: list[4]) as Any) as! FlutterStandardTypedData? 
+    let aNullable4ByteArray = (nilOrValue(value: list[5]) as Any) as! FlutterStandardTypedData? 
+    let aNullable8ByteArray = (nilOrValue(value: list[6]) as Any) as! FlutterStandardTypedData? 
+    let aNullableFloatArray = (nilOrValue(value: list[7]) as Any) as! FlutterStandardTypedData? 
+    let aNullableList = (nilOrValue(value: list[8]) as Any) as! [Any]? 
+    let aNullableMap = (nilOrValue(value: list[9]) as Any) as! [AnyHashable: Any]? 
+    let nullableNestedList = (nilOrValue(value: list[10]) as Any) as! [[Bool?]?]? 
+    let nullableMapWithAnnotations = (nilOrValue(value: list[11]) as Any) as! [String?: String?]? 
+    let nullableMapWithObject = (nilOrValue(value: list[12]) as Any) as! [String?: Any?]? 
```

Comment:
the `as Any` part is not necessary.

Reply:
these tests fail without it

https://github.com/flutter/packages/pull/3545/files#diff-2f49a83a24eeebd3ea0505c4bf30678b0dabbe79aa6f7eef4b48855a23b4d5ce

Reply:
I'm going to adjust the tests to fix this issue instead of leaving this as is though.